web/src/main/webapp/xslt/ui-metadata/form-builder.xsl

```diff
@@ -1299,15 +1301,17 @@
   <xsl:template name="render-form-field-control-remove">
     <xsl:param name="editInfo"/>
     <xsl:param name="parentEditInfo" required="no"/>
+    <xsl:param name="isRequired" required="no"/>
     <xsl:if
-      test="($parentEditInfo and
-                     ($parentEditInfo/@del = 'true' or
-                     $parentEditInfo/@min != 1)
-                   ) or
-                   (not($parentEditInfo) and ($editInfo and
-                   ($editInfo/@del = 'true' or
-                   $editInfo/@min != 1)
-                   ))">
+      test="(($parentEditInfo and (
+              $parentEditInfo/@del = 'true' or
+              $parentEditInfo/@min != 1)
+            ) or (
+              not($parentEditInfo) and $editInfo and (
+                $editInfo/@del = 'true' or
+                $editInfo/@min != 1
+              )
+            )) and not($isRequired and ($editInfo and $editInfo/@max and $editInfo/@max = 1) and ($parentEditInfo and $parentEditInfo/@max and $parentEditInfo/@max = 1))">
```

Comment:
The conditions ($editInfo and $editInfo/@max and $editInfo/@max = 1) and ($parentEditInfo and $parentEditInfo/@max and $parentEditInfo/@max = 1) are between parentesis for readability, because it's not possible to check directly $elem/@max = 1 without getting xpath errors.